build/yamls/antrea-aks.yml

```diff
@@ -1199,9 +1199,10 @@ data:
     # Enable metrics exposure via Prometheus. Initializes Prometheus metrics listener.
     #enablePrometheusMetrics: false
 
-    # Provide flow collector address as string with format <IP>:<port>[:<proto>], where proto is tcp or udp. This also enables
-    # the flow exporter that sends IPFIX flow records of conntrack flows on OVS bridge. If no L4 transport proto is given,
-    # we consider tcp as default.
+    # Provide flow collector address as string with format <IP>:<port>[:<proto>], where proto is tcp or udp. IP can be IPv4
+    # or IPv6 and IPv6 address should be wrapped with [].
```

Comment:
IP can be either IPv4 or IPv6; however, IPv6 address should be wrapped with [].

Reply:
Updated.